react/login.tsx

```diff
@@ -1,25 +1,29 @@
-import React from "react";
+import React, { useState } from "react";
 
 import useLogin from '/react/api/useLoggin';
 
 import Button from '.'
 import Input from '.'
 
 function Login() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
 
   return (<div>
     <form>
       <lable>username</lable>
       <Input 
-        value={'username'} 
-        onChange={() => {}}
-        placeholder="Please enter your username."
+        value={username} 
+        onChange={(e) => { setUsername(e.target.value) }}
```

Comment:
Since I initially believed the input tag was a built-in HTML feature, I didn't realize I could simply write it like this: onChange={(username) => setUsername(username)}. The same thing applies to the password input.